packages/typespec-test/test/faceai/generated/typespec-ts/README.md

```diff
@@ -0,0 +1,57 @@
+# Azure Face REST client library for JavaScript
```

Comment:
I put face API into smoke-test as there's multipart-form data related change. so I guess it's okay.